tests/rptest/tests/partition_movement.py

```diff
@@ -114,6 +122,9 @@ def status_done():
                     f"current assignments for {topic}-{partition}: {info}")
                 converged = self._equal_assignments(info["replicas"],
                                                     assignments)
+                if previous_assignmets is not None:
```

Comment:
this is wrong, we check if it's A or B per node so this method may finish but different nodes will have different configurations